cves/CVE-2011-3190.yml

```diff
@@ -10,27 +11,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: 264
```

Comment:
If the issue is really input validation, then is 264 the best fit?  Maybe https://cwe.mitre.org/data/definitions/20.html, https://cwe.mitre.org/data/definitions/88.html, or https://cwe.mitre.org/data/definitions/94.html?